sycl/include/sycl/ext/oneapi/experimental/cluster_group_prop.hpp

```diff
@@ -0,0 +1,79 @@
+//==--- cluster_group_prop.hpp --- SYCL extension for cluster group
+//------------------==//
```

Comment:
Please fix this part of the header to be a single line.

Reply:
addressed in [f0f9b](https://github.com/intel/llvm/pull/14113/commits/f0f9bfdcfb2b98b0deb633aff547f5f014b1d0a5)